stable/traefik/README.md

```diff
@@ -117,6 +117,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `dashboard.statistics.recentErrors` | Number of recent errors to show in the ‘Health’ tab              | None                                      |
 | `service.annotations`           | Annotations for the Traefik Service definition, specified as a map   | None                                      |
 | `service.labels`                | Additional labels for the Traefik Service definition, specified as a map | None                                  |
+| `service.nodePorts.http`           | Desired nodePort for service of type NodePort used for http requests  |   dynamic nodePort allocation                                      |
+| `service.nodePorts.https`           | Desired nodePort for service of type NodePort used for https requests  | dynamic nodePort allocation                                      |
```

Comment:
I still think that for both of these it should be made more clear that not set == dynamic allocation. i.e. The description indicates the value should be a port number, but the default value is not a number. It will not be obvious to everyone who reads this that the default is "dynamic nodePort allocation" how that was achieved.

Reply:
should I say " blank ('') - will assign a dynamic node port "